integration/test_usercontext.py

```diff
@@ -0,0 +1,3 @@
+def test_get_current_tenant(connection):
+    response = connection.usercontext.get_current_tenant_id()
+    assert response == "1d71796f-af5b-4231-9d8e-df6434da4663"
```

Comment:
All though this is a test system, Does exposing the `tenant-id` have any risk, given that Py42 is a publicly accessible?

This endpoint does not return `Py42Response`.

Reply:
You could do

```
assert type(response) == str
assert len(response) == 36
```

That should cover any tenant ID

Reply:
Changed.